ui/documents/semantic-ui/templates/documents/form/FormFieldsContainer.jsx

```diff
@@ -185,19 +186,24 @@ const FormFieldsContainer = () => {
             id="NrDocs.Deposit.LicenseField.container"
             fieldPath="metadata.rights"
           >
-            <LocalVocabularySelectField
-              optimized
+            <LicenseField
+              searchConfig={{
+                searchApi: {
+                  axios: {
+                    headers: {
+                      Accept: "application/json",
+                    },
+                    url: "/api/vocabularies/rights",
+                    withCredentials: false,
```

Comment:
This breaks if we decide to impose authn on that api

Reply:
@mirekys You are right, sorry, it was a copy paste from rdm's component.